lib/node_modules/@stdlib/utils/error-to-json/README.md

```diff
@@ -0,0 +1,243 @@
+# Error to JSON
```

Comment:
A bit of an awkward title. Sense a better title would be to match the function name.